app/assets/stylesheets/homepage.css.scss

```diff
@@ -20,7 +20,7 @@
 #nav-bar h1 {
     font-family: 'Yellowtail', cursive;
     font-size: 48px;
-    padding-left: 15%;
+    padding-left: 19%;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered font-family, font-size, font-weight, padding-left